pages/common/pnmquant.md

```diff
@@ -0,0 +1,9 @@
+# pnmquant
+
+> Quantize the colors in a PNM image into a smaller set.
+> This command is a combination of `pnmcolormap` and `pnmremap` and accepts the union of their options.
```

Comment:
pnmquant is literally a combination of pnmcolormap and pnmremap, and as such accepts all options of these two commands (except -mapfile).
That's why I did not add many examples, since they would be redundant. 
Do y'all think this is okay?

Reply:
Done.